lang/cs/Org.Apache.REEF.Tests/Org.Apache.REEF.Tests.csproj

```diff
@@ -67,6 +67,9 @@ under the License.
     <Compile Include="Utility\TestExceptions.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="ConfigFiles\evaluator.conf">
```

Comment:
This file strikes me as one we could auto-generate in the build in the future. Correct? If so, we should file a JIRA for that change.